ui/app/AppLayouts/Onboarding2/KeycardCreateReplacementFlow.qml

```diff
@@ -102,12 +105,14 @@ SQUtils.QObject {
         id: keycardCreatePinPage
 
         KeycardCreatePinPage {
+            pinSettingState: root.pinSettingState
+            authorizationState: root.authorizationState
             onKeycardPinCreated: (pin) => {
-                Backpressure.debounce(root, root.keycardPinInfoPageDelay, () => {
```

Comment:
debounce removed on purpose here?

Reply:
To display the "Keycard PIN set" page with the success info, and then emit the signal and move forward; otherwise that page would just blink in front of the user's eyes

Reply:
@caybro For this purpose there's `debounce` inside `KeycardCreatePinPage`:
https://github.com/status-im/status-desktop/blob/be1059b827c7fa3bdffd43ee255a3489af7a6d51/ui/app/AppLayouts/Onboarding2/pages/KeycardCreatePinPage.qml#L179-L182